R/civis_ml_utils.R

```diff
@@ -163,6 +163,26 @@ get_metric <- function(model, name = NULL) {
   }
 }
 
+
+#' Get model feature importance
+#' @param model Object from \code{\link{civis_ml}}
+#' @return A matrix of features and their importance, ranked in descending order of importance
+#' @export
+get_feature_importance <- function(model){
+
+  if (is.null(model$metrics$model$parameters$feature_importances)) stop("Feature importance data not available.")
```

Comment:
I might declare a variable here

```r
params <- model$metrics$model$parameters
```

To avoid having to repeat `model$metrics$model$parameters` below